WordPress/Classes/Models/Blog.h

```diff
@@ -60,17 +60,24 @@ typedef NS_ENUM(NSUInteger, BlogFeature) {
 @property (nonatomic, strong, readwrite) WPAccount      *jetpackAccount;
 @property (nonatomic, assign, readwrite) BOOL           videoPressEnabled;
 @property (nonatomic, assign, readwrite) BOOL           isMultiAuthor;
-@property (nonatomic, assign, readwrite) BOOL           isJetpack;
+@property (nonatomic, assign, readwrite) BOOL           isHostedAtWPcom;
 @property (nonatomic, strong, readwrite) NSString       *icon;
+/**
+ Stores the username for self hosted sites
+ 
+ @warn For WordPress.com or Jetpack Managed sites this will be nil. Use usernameForSite instead
+ */
+@property (nonatomic, strong,  readwrite) NSString       *username;
```

Comment:
I'm not a big fan of the style either, but it seems @jleandroperez likes to align things
https://github.com/wordpress-mobile/WordPress-iOS/commit/b629aae5e6ba2c3dfb6d41b18390f8fca0db8b4b